### PR TITLE
vim-patch:partial 2346a6378483

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/2346a6378483c9871016f9fc821ec5cbea638f13

omit doc/vim.man
omit doc/vim9.txt
omit doc/vimdiff.txt
omit doc/vimdiff.man

partial skip doc/eval.txt (missing patches 8.2.2324, 8.2.2344)

### CODE REVIEW

**Review by justinmk — APPROVED** [review]

**Comment by clason** [comment]
@justinmk general question: would it be better to merge this now and add the skipped hunks either when the actual prerequisite patches are merged (or in a big mop-up operation close to next release) or park this until someone ports these patches?

(Of course, @janlazo would have added the patches to this PR, but... I'm not janlazo 😞 )

**Comment by justinmk** [comment]
> general question: would it be better to merge this now and add the skipped hunks either when the actual prerequisite patches are merged

If it's a subtle tweak to the docs I usually just include it now, even if it's invalid. Hard to followup on that later.

If it's a new function/command/etc it will be obvious that it's missing later and will eventually get added/reported. No need to track that or worry about it. Could also just optimistically include it here now.

**Comment by clason** [comment]
It's the latter: doc fixes (very, very minorly) to new functions introduced in the mentioned patches.
